lib/index.js

```diff
@@ -19,7 +19,9 @@ function nodeNetCatSrc(port, input) {
   );
 }
 
-const FUNCTION_PRIORITY = [nativeNC, nodeNC];
+// Use nodeNC to avoid native nc error (spawnSync error observed with msc os 11.2.3)
```

Comment:
@c3wenjiaowang  is there a spawnSync issue that you can link here as well (one that describes OS bug) or its not online and you just happened to try it /  notice it

Reply:
@c3-tiffanyhui Didn't find any issue reported online, just figured with the 3 colleagues who reported this issue, and they are all on mac os 11.2.3...